BunqSdk/Exception/ApiException.cs

```diff
@@ -1,32 +1,22 @@
-﻿using System.Collections.Generic;
-
+﻿
 namespace Bunq.Sdk.Exception
 {
-    /// <summary>
-    /// Exception triggered by API requests failed on the server side.
-    /// </summary>
     public class ApiException : System.Exception
     {
-        /// <summary>
-        /// Glue to concatenate the error messages.
-        /// </summary>
-        private const string GLUE_ERROR_MESSAGES = "\n";
-
+        private readonly string message;
         public int ResponseCode { get; private set; }
-        public IList<string> Messages { get; private set; }
 
-        /// <param name="responseCode">The HTTP Response code of the failed request.</param>
-        /// <param name="messages">The list of messages related to this exception.</param>
-        public ApiException(int responseCode, IList<string> messages) : base(
-            ConcatenateMessages(messages))
+        public override string Message
         {
-            ResponseCode = responseCode;
-            Messages = messages;
+            get { return message; }
```

Comment:
If we opt for using the message after all, you would need to do { get; private set} here :)

Reply:
We can't override the `set` method because the base doesn't have a set method to override tho 🤔 So guess it would need to be a local method.